packages/config/src/index.ts

```diff
@@ -10,6 +11,9 @@ declare global {
   }
 }
 
-export const Environment: EnvironmentConfig = window.env ?? {
-  API_URL: 'http://localhost:4000',
+const { API_URL = 'http://localhost:4000' } = window.env;
```

Comment:
The default here won't work with the replace down there will it?

I wonder if this API_URL is better though?

```
interface EnvironmentConfig {
  API_URL: string;
  GRAPHQL_URL: string;
  FILE_URL: string;
}
...
const { API_URL = 'http://localhost:4000' } = window.env;

export const Environment: EnvironmentConfig = {
  API_URL,
   GRAPHQL_URL: `${Environment.API_URL}/graphql}`
  FILE_URL: `${Environment.API_URL/files?id=}`,
};
```

Reply:
ah. I'm going to miss your little bursts of enlightenment. that's much betterer.
`API_ROOT` maybe?